website/.gitignore

```diff
@@ -1,4 +1,3 @@
-node_modules
```

Comment:
I meant the whole file, why not just stick the ignores you need in the root `.gitignore`?

Reply:
@SimenB Ahh, I see. In event we ever break out the website into its own repo. For time being, I'll migrate the paths to the top level `.gitignore`.